tests/test_auto_lambda.py

```diff
@@ -65,9 +51,9 @@ def test_dlambda(self):
                               store_observables=True)
 
         sg_comp = SpectrumGenerator(
-            lambda_min=sg_auto.lambda_field[0].d,
```

Comment:
All of these changes are because `SpectrumGenerator` now allows either floats or `YTQuantities` where previously it only allows floats.